crates/sui-protocol-config/src/lib.rs

```diff
@@ -117,6 +118,8 @@ struct FeatureFlags {
     // If true, validators will commit to the root state digest
     // in end of epoch checkpoint proposals
     commit_root_state_digest: bool,
+    // Allow conservation checks when true
+    conservation_checks: bool,
```

Comment:
If this is something we might want to have on in a full node but not in prod, is protocol config the right mechanism (tagging @longbowlu for this)

Reply:
good question, I'll follow up. I think we want the config and we can always compile with the config on (as opposed to other form of compilation). But if there was an easy way to turn it on/off on a full node that would be wonderful